docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-potential-lsass-clone-creation-via-psscapturesnapshot.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[prebuilt-rule-0-16-2-potential-lsass-clone-creation-via-psscapturesnapshot]]
+=== Potential LSASS Clone Creation via PssCaptureSnapShot
+
+Identifies the creation of an LSASS process clone via PssCaptureSnapShot where the parent process is the initial LSASS process instance. This may indicate an attempt to evade detection and dump LSASS memory for credential access.
```

Comment:
```suggestion
Identifies the creation of a Local Security Authority Server Service (LSASS) process clone via PssCaptureSnapShot where the parent process is the initial LSASS process instance. This may indicate an attempt to evade detection and dump LSASS memory for credential access.
```